frameworks/remix-run/application/src/routes/$langstore.orders.tsx

```diff
@@ -10,10 +10,14 @@ export const headers: HeadersFunction = ({ loaderHeaders }) => {
 };
 
 export const loader: LoaderFunction = async ({ request }) => {
```

Comment:
`useLoaderData<typeof loader>();` + change the loader using `LoaderFunctionArgs`